packages/bitcoin-networks/lib/index.ts

```diff
@@ -30,4 +70,158 @@ const BitcoinNetworks = {
   bitcoin_regtest
 }
 
-export { BitcoinNetworks }
+export interface BitcoinCashNetwork extends BitcoinNetwork {
+  prefix: string
+}
+
+const bitcoin_cash: BitcoinCashNetwork = {
+  name: 'bitcoin_cash',
+  protocolType: ProtocolType.BitcoinCash,
+  prefix: 'bitcoincash',
+  feeBumpCapable: false,
+  segwitCapable: false,
+  usePSBT: false,
+  addrToBitcoinJS: infallibleTransformToBitcoin(cashaddrToLegacy),
+  addrFromBitcoinJS: infallibleTransformFromBitcoin(legacyToCashAddr),
+  messagePrefix: '\x18Bitcoin Signed Message:\n',
+  bech32: '',
+  bip32: {
+    // Networks.mainnet.xpubkey
+    public: 0x0488b21e,
+    // Networks.mainnet.xprivkey
+    private: 0x0488ade4
+  },
+  pubKeyHash: 0x00,
+  scriptHash: 0x05,
+  // Networks.mainnet.privatekey
+  wif: 0x80,
+  coinType: '145',
+  isTestnet: false
+}
+
+const bitcoin_cash_testnet: BitcoinCashNetwork = {
+  name: 'bitcoin_cash_testnet',
+  protocolType: ProtocolType.BitcoinCash,
+  prefix: 'bchtest',
+  feeBumpCapable: false,
+  segwitCapable: false,
+  usePSBT: false,
+  addrToBitcoinJS: infallibleTransformToBitcoin(cashaddrToLegacy),
+  addrFromBitcoinJS: infallibleTransformFromBitcoin(legacyToCashAddr),
+  messagePrefix: '\x18Bitcoin Signed Message:\n',
+  bech32: '',
+  bip32: {
+    public: 0x043587cf,
+    private: 0x04358394
+  },
+  pubKeyHash: 0x6f,
+  scriptHash: 0xc4,
+  wif: 0xef,
+  coinType: '146',
+  isTestnet: true
+}
+
+const bitcoin_cash_regtest: BitcoinCashNetwork = {
+  name: 'bitcoin_cash_regtest',
+  protocolType: ProtocolType.BitcoinCash,
+  prefix: 'bchreg',
+  feeBumpCapable: false,
+  segwitCapable: false,
+  usePSBT: false,
+  addrToBitcoinJS: infallibleTransformToBitcoin(cashaddrToLegacy),
+  addrFromBitcoinJS: infallibleTransformFromBitcoin(legacyToCashAddr),
+  messagePrefix: '\x18Bitcoin Signed Message:\n',
+  bech32: '',
+  bip32: {
+    public: 0x043587cf,
+    private: 0x04358394
+  },
+  pubKeyHash: 0x6f,
+  scriptHash: 0xc4,
+  wif: 0xef,
+  coinType: '146',
+  isTestnet: true
+}
+
+const BitcoinCashNetworks = {
+  bitcoin_cash,
+  bitcoin_cash_testnet,
+  bitcoin_cash_regtest
+}
+
+export { BitcoinNetworks, BitcoinCashNetworks }
+
+function infallibleTransformFromBitcoin(transformer: (a: string, b: BitcoinNetwork) => string) {
+  return function (address: string, network: BitcoinNetwork) {
+    try {
+      return transformer(address, network)
+    } catch (e) {
+      return address
+    }
+  }
+}
+
+function infallibleTransformToBitcoin(transformer: (a: string) => string) {
+  return function (address: string) {
+    try {
+      return transformer(address)
+    } catch (e) {
+      return address
+    }
+  }
+}
+
+// Bitcoin Cash
+
+function cashaddrToLegacy(address: string) {
```

Comment:
We like to keep the BitcoinNetwork class serializable as it gets passed between contexts sometimes

Is this format only for bitcoin cash? In that case it would be great if it could be moved to `bitcoin-utils` instead. 

something like 

```
addToBitcoinJs (address: string, network: BitcoinNetwork) {
 if(!bitcoincash) return address

  ....
 // handle the cash address format
}
```